src/background/messaging/rpc-message-handler.ts

```diff
@@ -22,6 +23,10 @@ export async function rpcMessageHandler(message: WalletRequests, port: chrome.ru
       await rpcOpen(message, port);
       break;
     }
+    case 'swap': {
```

Comment:
Would call it this message something less terse, to indicate that this isn't our API to initiate/track a swap, rather, just opening that page.